src/spatio_temporal_voxel_layer.cpp

```diff
@@ -636,6 +636,44 @@ void SpatioTemporalVoxelLayer::updateCosts( \
   return;
 }
 
+/*****************************************************************************/
+void SpatioTemporalVoxelLayer::clearArea(int start_x, int start_y, int end_x, int end_y)
+/*****************************************************************************/
+{
+  //easy way reset the whole grid.
+  _voxel_grid->ResetGrid();
+  Costmap2D::resetMaps();
+
+
+  // hard way selected points
```

Comment:
In general, I dont like to commit commented out code into a master branch

Reply:
Sure I will remove it. It was an easy way of sharing my thoughts.